controllers/operator/configmap.go

```diff
@@ -22,6 +22,7 @@ import (
 	"crypto/tls"
 	"crypto/x509"
 	"encoding/json"
+	myerr "errors"
```

Comment:
Would you please instead avoid naming this import and name the `k8s.io/apimachinery/pkg/api/errors` package "k8sErrors" - this is what I've been doing in other files whenever something like this comes up.

Reply:
sure